superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -852,25 +852,33 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
     setTabKey(key);
   };
 
-  const renderStepTwoAlert = () =>
-    db?.engine && (
-      <StyledAlertMargin>
-        <Alert
-          closable={false}
-          css={(theme: SupersetTheme) => antDAlertStyles(theme)}
-          type="info"
-          showIcon
-          message={
-            engineSpecificAlertMapping[db.engine]?.message ||
-            connectionAlert?.DEFAULT?.message
-          }
-          description={
-            engineSpecificAlertMapping[db.engine]?.description ||
-            connectionAlert?.DEFAULT?.description
-          }
-        />
-      </StyledAlertMargin>
+  const renderStepTwoAlert = () => {
+    const path = window.location.hostname;
+    const region = path.split('.')[1];
+    const ipAlert =
+      connectionAlert?.REGIONAL_IPS[region] ||
+      connectionAlert?.REGIONAL_IPS?.us1a;
```

Comment:
Unfortunately this logic will only work for Preset, we need to come up with a more generic way of doing this.

Reply:
I think this alert only exists in preset though.

Reply:
Even if it currently only exists in Preset, we shouldn't have Preset-specific logic here. Other users might want to define an alert for their systems using the same scaffolding we use for Preset, and their hostnames probably follow a different convention.

A more generic pattern is to have the keys be regular expressions, and we test it against the whole hostname.